config/custom-environment-variables.yaml

```diff
@@ -21,6 +21,8 @@ auth:
     # A flag to set if the server is running over https.
     # Used as a flag for the OAuth flow
     https: IS_HTTPS
+    # A flag to set if you want guests to browse your pipelines
+    allowGuestAccess: SECRET_GUEST_ACCESS
```

Comment:
maybe just `GUEST_ACCESS` since it's just a flag. Secret makes it sounds like a password.

Reply:
That's fair, I was just following the previous standard in this section.  The original name of the group was "secret" prior to being renamed to "auth".  I would prefer `AUTH_GUEST_ACCESS` instead.

Reply:
sure. that works for me.

Reply:
Fixed!